### ISSUE TITLE
2H/BNOS Standard Computation

### ISSUE THREAD

**Issue by mtcuison** [mtcuison/GMC-Integrated-Systems#5]
- [x] Encode to database the rates
- [x] Adjust ggcPriceUpdate.clsMCPricelist computation
- [ ] Create 2H/BNOS Installment Calculator

**Comment by szirabanal** [comment]
is this case same with BNOS case #4 ?
How to check the computation? is there another form/parameter po?

**Comment by mtcuison** [comment]
> is this case same with BNOS case #4 ? How to check the computation? is there another form/parameter po?

yes. this is connected to that issue. created a separate issue since the objects that are modified will affect 2H and BNOS computation and will also need to include re-testing of Brand New installment computation to be sure.
